src/main/kotlin/dataSource/CsvDataSource.kt

```diff
@@ -24,6 +24,10 @@ class CsvDataSource(private val parser: CsvParser,private val file :String= FILE
 
     companion object {
         private const val FILE_NAME = "csvFiles/costOfLiving.csv"
+        private const val SUB_FILE_NAME = "csvFiles/subListOfCostOfLiving.csv"
```

Comment:
why are you declaring these variables?

1. SUB_FILE_NAME
2.  SUB_FILE_NAME
3.  ONE_CITY_FILE_NAME
4. EMPTY_FILE_NAME

They're not used anywhere in your branch.
- [ ] delete them all

Reply:
i forgot to remove them..